### PR TITLE
add props for European cities carousel

### PR DESCRIPTION
**Pull request by CharlyJuteau**

### CODE REVIEW

**Review by PriscilliaAmmeux — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+  "keywords": [],
+  "author": "Wild Code School",
+  "license": "ISC",
+  "devDependencies": {
+    "concurrently": "^7.0.0",
+    "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "prop-types": "^15.8.1",
+    "react-leaflet": "^4.1.0"
```
Comment (LoicBrassart) [inline package.json]: Not there

File: package.json [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
+  },
+  "keywords": [],
+  "author": "Wild Code School",
+  "license": "ISC",
+  "devDependencies": {
+    "concurrently": "^7.0.0",
+    "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "prop-types": "^15.8.1",
```
Comment (LoicBrassart) [inline package.json]: Not there

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import "./reset.css";
+// eslint-disable-next-line import/no-named-as-default
```
Comment (LoicBrassart) [inline frontend/src/App.jsx]: Stop disabling ESLint rules: learn them!

File: frontend/src/components/Carousel.jsx [hunk 4]
```diff
@@ -1,5 +1,6 @@
+/* eslint-disable no-shadow */
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel.jsx]: Stop this

File: frontend/src/components/Carousel.jsx [hunk 5]
```diff
@@ -1,5 +1,6 @@
...
 import { Swiper, SwiperSlide } from "swiper/react";
-
+import { PropTypes } from "prop-types";
 // eslint-disable-next-line import/no-unresolved
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel.jsx]: // eslint-disable import/no-unresolved
import "swiper/css";
import "swiper/css/pagination";
import "swiper/css/navigation";
// eslint-enable import/no-unresolved

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
SOme Evils to fix

File: frontend/src/components/Carousel.jsx (hunk 5 above)
Comment (LoicBrassart) [inline frontend/src/components/Carousel.jsx]: Told ya already how to write this

File: frontend/src/components/Carousel.jsx [hunk 6]
```diff
@@ -35,69 +36,33 @@ export default function Carousel() {
...
+              <p className="CitiesName">{EuropeanCities.cityName} </p>
+            </SwiperSlide>
+          </div>
+        );
+      })}
     </Swiper>
   );
 }
+
+Carousel.propTypes = {
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel.jsx]: Nice :ok_hand:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please stop disabling ESLint when unnecessary :p

**Comment by LoicBrassart** [comment]
Missing SM validation, plrease reassign me later

**Review by Azahires — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
Ongoing conflicts, please reassign me later

**Review by cartige — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 7]
```diff
@@ -1,7 +1,8 @@
 import "./reset.css";
-import Carousel from "./components/Carousel";
+import Carousel from "./components/Carousel/Carousel";
 import "./App.css";
 import Header from "./components/Header/Header";
+// eslint-disable-next-line import/order, import/no-unresolved
```
Comment (LoicBrassart) [inline frontend/src/App.jsx]: No

File: frontend/src/pages/Activities/Activities.jsx [hunk 8]
```diff
@@ -0,0 +1,80 @@
+/* eslint-disable import/no-unresolved, import/extensions */
+import "./reset.css";
+import Carousel from "./frontend/src/components/Carousel/Carousel";
```
Comment (LoicBrassart) [inline frontend/src/pages/Activities/Activities.jsx]: Wait, this shouldn't work ಠ_ಠ

From the `frontend/src/pages/Activities` folder, you ask `./frontend/[...]`
This'd mean you're searching your component in `frontend/src/pages/Activities/frontend/[...]`, and this folder doesn't exist, does it ? 
_(NB: that's an example of why I don't like when you disable ESLint rules, I'm not sure how the application behaves)_

-> If it works, no "need" to rewrite, but it'd be better, really

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One (recurring) Evil to fix + a wtf moment for me !

**Review by cartige — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil !

**Pull request merged**